### DISCUSSION TITLE
my app doesn

### DISCUSSION THREAD

**Discussion by elikent** [Budibase/budibase#5183] [Help] [answered]
Not sure what I'm doing wrong. Any help would be appreciated.

**What is happening**
I'm developing a simple two-screen app.
Screen 1 is called "Miembros-de-la-familia" and contains a form.
Screen 2 is called "Thank-you-page" and is just a thank you message
Both screens are set to public. 
I published the app as is. But when I go to "View published app" I see three screens, all with clones of my "Miembros-de-la-familia" screen.
Screenshots below. 

**What I tried**
I deleted cookies and chache. I tried a different web browser (Chrome and Edge). Still same issue

Any ideas?
Screen shots below.

**Design page**
![image](https://user-images.githubusercontent.com/94906328/160886808-1a33e9bd-c26f-465d-b678-6e96018203ef.png)

**Public app**
![image](https://user-images.githubusercontent.com/94906328/160886873-76eb219f-8f10-4f41-9615-ad336f1cd7eb.png)

**Comment by mike12345567** [comment, marked as answer]
Hi @elikent - I have converted this to a discussion as it is a support query.

In your case it appears you have created two screens, can you delete one of them - either the `HDLN - Thank you` or the `PRGF - Thank you`, which ever one is not needed.

As for the three screens in your nav, this is the default behaviouir of the navigation to provide a home nav button, you can remove this by going to your layout, and clicking the configure links button.
![image](https://user-images.githubusercontent.com/4407001/160893585-71f796ff-1425-4210-95e4-3713c8a9e644.png)

This brings up a drawer where you can remove the Home link.
![image](https://user-images.githubusercontent.com/4407001/160893648-e1ea6ff5-2bbe-4957-96d7-28100720efc3.png)

**Reply by elikent** [reply]
Oh my! Mike, thank you so much. Solved this problem and also helped me understand what layout does.
